### DISCUSSION TITLE
Pinia does not appear to work with Vue 2.7

### DISCUSSION THREAD

**Discussion by colinblaise** [vuejs/pinia#1495] [Help and Questions] [answered]
### Reproduction

todo

### Steps to reproduce the bug

Create a new vue project with 2.7 and add Pinia.

### Expected behavior

Should be able to run vite dev without issues.

### Actual behavior
```
Error: Build failed with 1 error:
node_modules/vue-demi/lib/index.mjs:2:30: ERROR: Could not resolve "@vue/composition-api/dist/vue-composition-api.mjs"
 
  errors: [
    {
      detail: undefined,
      id: '',
      location: {
        column: 30,
        file: 'node_modules/vue-demi/lib/index.mjs',
        length: 51,
        line: 2,
        lineText: "import VueCompositionAPI from '@vue/composition-api/dist/vue-composition-api.mjs'",
        namespace: '',
        suggestion: ''
      },
      notes: [
        {
          location: null,
          text: 'You can mark the path "@vue/composition-api/dist/vue-composition-api.mjs" as external to exclude it from the bundle, which will remove this error.'
        }
      ],
      pluginName: '',
      text: 'Could not resolve "@vue/composition-api/dist/vue-composition-api.mjs"'
    }
  ],
  warnings: []
}
```

**Comment by colinblaise** [comment]
Duplicate https://github.com/vuejs/pinia/issues/1417

**Comment by colinblaise** [comment]
Reopening because I actually still can't figure out what the correct path forward is.

Am I still required to manually include @vue/composition-api in my package.json if I'm using Pinia and Vue 2.7?

**Comment by posva** [comment, marked as answer]
Make sure to upgrade vue-demi too! (https://github.com/vuejs/pinia/discussions/1380#discussioncomment-3139267)

**Comment by colinblaise** [comment]
Womp, of course it says this directly in https://blog.vuejs.org/posts/vue-2-7-naruto.html.  Much appreciated!
